src/Apps/Artist/Components/ArtistHeader/ArtistHeaderImage.tsx

```diff
@@ -0,0 +1,57 @@
+import { BoxProps, Image, ResponsiveBox } from "@artsy/palette"
+import { FC } from "react"
+import { maxDimensionsByArea, resized } from "Utils/resized"
+
+interface ArtistHeaderImageProps
+  extends Omit<BoxProps, "maxHeight" | "maxWidth"> {
+  image: ValidImage
+}
+
+export const ArtistHeaderImage: FC<ArtistHeaderImageProps> = ({
+  image,
+  ...rest
+}) => {
+  const max = maxDimensionsByArea({
+    width: image.width,
+    height: image.height,
+    area: 300 * 300,
+  })
+
+  const img = resized(image.src, { width: max.width, height: max.height })
+
+  return (
+    <ResponsiveBox
+      aspectWidth={image.width}
+      aspectHeight={image.height}
+      maxWidth={max.width}
+      maxHeight={max.height}
+      {...rest}
+    >
+      <Image
+        src={img.src}
+        srcSet={img.srcSet}
+        width="100%"
+        height="100%"
+        style={{ objectFit: "cover" }}
+      />
+    </ResponsiveBox>
+  )
+}
+
+interface ValidImage {
+  src: string
+  width: number
+  height: number
+}
+
+interface MaybeValidImage {
+  src?: string | null
+  width?: number | null
+  height?: number | null
+}
+
+export const isValidImage = (
```

Comment:
Curious how this is being used? I think i know but wondering what it looks like in the code as it seems handy

Reply:
I'm not sure, but this looks like it performs some kind of type guard behavior to narrow down the chaos (lol) to make sure that the image meets the valid criteria necessary for it to be  considered a validImage??

Reply:
if isValidImage() is true then typescript knows the image is valid. Then you can safely access the src, width and height properties on it, right?